Mathlib/Topology/UniformSpace/Basic.lean

```diff
@@ -1566,8 +1568,8 @@ example [UniformSpace α] [UniformSpace β] :
 
 theorem uniformity_prod [UniformSpace α] [UniformSpace β] :
     𝓤 (α × β) =
-      ((𝓤 α).comap fun p : (α × β) × α × β => (p.1.1, p.2.1)) ⊓
-        (𝓤 β).comap fun p : (α × β) × α × β => (p.1.2, p.2.2) :=
+      ((𝓤 α).comap fun ((a₁, _), (a₂, _)) ↦ (a₁, a₂)) ⊓
+        (𝓤 β).comap fun ((_, b₁), (_, b₂)) ↦ (b₁, b₂) :=
```

Comment:
I think this new spelling is worse; while the source may be more readable the generated `Expr` is much more verbose, as it now contains `match` statements (aka `uniformity_prod._match_1` and `uniformity_prod._match_2`)

Reply:
I've reverted to the former spelling. The only one for which I've kept the new spelling is the (new) definition `UniformityProd`, which, if I'm not mistaken, doesn't generate anything strange.

That's a good lesson to always check the generated expressions; I'll keep that in mind!